Rubberduck.Core/app.config

```diff
@@ -260,160 +260,6 @@
           </AutoCompleteSettings>
         </value>
       </setting>
-      <setting name="CodeInspectionSettings" serializeAs="Xml">
```

Comment:
Was blowing away the default CodeInspectionSettings intentional?

Reply:
Absolutely not, LOL.